### PR TITLE
Update diagnose-problems-with-the-skype-for-business-online-connector.md

### PR DESCRIPTION
**Pull request by tomkau**
Adding information about the need to enable WinRM for Basic authentication.  

We had a customer create a group policy that disabled WinRM for their computers and they were no longer able to execute SfBO PS scripts.  They need this documented so they can get an exception from their IT for some of their computers to allow basic auth

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
@tonysmit  Changes look correct. Could you please review and merge? Thanks!

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md [hunk 1]
```diff
@@ -64,6 +66,13 @@ The Skype for Business Online Connector module can be run only under Windows Pow
 
 - **Resolution**: The only way to fix this problem is to install Windows PowerShell 3.0, which is available from the Microsoft Download Center at [https://www.microsoft.com/en-us/download/details.aspx?id=34595](https://www.microsoft.com/en-us/download/details.aspx?id=34595).
   
+## Modern authentication fails when WinRM Basic authentication has been disabled
+<a name="BKMKWinRMBasicAuth"> </a>
+
+The latest version of the Skype for Business Online Connector module uses modern authentication, but the underlying Windows Remote Management (WinRM) client must be configured to allow Basic authentication.  Modern authentication uses bearer tokens which are usually passed in the *Authorization: Bearer* header, but Windows PowerShell upon which Skype for Business PowerShell is built does not allow for manipulation of this header.   Instead, Skype for Business PowerShell uses the *Authorization: Basic* header to pass the bearer token.
```
Comment (nenonix) [inline Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md]:
```suggestion
The latest version of the Skype for Business Online Connector module uses modern authentication, but the underlying Windows Remote Management (WinRM) client must be configured to allow Basic authentication.  Modern authentication uses bearer tokens which are usually passed in the *Authorization: Bearer* header. Windows PowerShell, upon which Skype for Business PowerShell is built, does not allow for manipulation of this header.  Instead, Skype for Business PowerShell uses the *Authorization: Basic* header to pass the bearer token.
```

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md [hunk 2]
```diff
@@ -64,6 +66,13 @@ The Skype for Business Online Connector module can be run only under Windows Pow
...
+
+See [Download and install Windows PowerShell](https://docs.microsoft.com/en-us/SkypeForBusiness/set-up-your-computer-for-windows-powershell/download-and-install-windows-powershell-5-1) for instructions on how to enable WinRM for Basic authentication.
```
Comment (nenonix) [inline Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md]:
```suggestion
See [Download and install Windows PowerShell](https://docs.microsoft.com/SkypeForBusiness/set-up-your-computer-for-windows-powershell/download-and-install-windows-powershell-5-1) for instructions on how to enable WinRM for Basic authentication.
```

File: Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md (hunk 1 above)
Comment (tomkau) [reply Skype/SfbOnline/set-up-your-computer-for-windows-powershell/diagnose-problems-with-the-skype-for-business-online-connector.md]: Made the suggested update.

**Comment by tomkau** [comment]
@tonysmit @scanum -- can this be merged?  A customer is waiting for this update so that they can get an exception from their IT team to enable Basic auth.

Thanks!

**Comment by scanum** [comment]
@tonysmit Changes look correct. Could you please review and merge? Thanks!

**Pull request merged**